discord/colour.py

```diff
@@ -53,6 +53,9 @@ class Colour:
         .. describe:: str(x)
 
              Returns the hex format for the colour.
+             
+        .. describe:: int(x)
+             Returns the raw color value.
```

Comment:
```suggestion
        .. describe:: int(x)

             Returns the raw color value.
```
Small nit to make the documentation appear properly

Reply:
Thanks, I guess I overlooked that.